ansible/build-vars.yml

```diff
@@ -0,0 +1,9 @@
+nodejs_app_name: universal
+nodejs_app_install_dir: /opt/{{ nodejs_app_name }}
```

Comment:
I've filed a bug about this in GPII's JIRA: https://issues.gpii.net/browse/GPII-1527